sentences/de/media_player_HassMediaPrevious.yaml

```diff
@@ -0,0 +1,15 @@
+language: de
+intents:
+  HassMediaPrevious:
+    data:
+      - sentences:
+          - "[<starte> ](vorherige|letzte)[r|s|n][ <song>][ (<an>|am)][ artikel_bestimmt] <name>"
```

Comment:
we should make `(vorherige|letzte)[r|s|n]` an expansion rule

Reply:
implemented